packages/playground/src/views/nodes.vue

```diff
@@ -546,18 +547,21 @@ export default {
       ipv6: false,
       myRentedNodes: false,
     });
-    const oldNodeStatus = ref();
-    watch(
-      () => filters.value.rentable,
-      rentable => {
-        if (rentable) {
-          oldNodeStatus.value = filters.value.status;
-          filters.value.status = UnifiedNodeStatus.UpStandby;
-        } else {
-          filters.value.status = oldNodeStatus.value;
-        }
-      },
-    );
+    const oldNodeStatus = computed(() => {
+      return filters.value.status;
+    });
```

Comment:
Doesn't truly preserve the previous value of status.

oldNodeStatus just mirrors filters.value.status, so when rentable is true and status is overridden, oldNodeStatus.value becomes UpStandby, and thus you lose the original.

this can be tested by selecting rentable you will see that the value of status field will be 
<img width="522" alt="image" src="https://github.com/user-attachments/assets/7ea092b0-b806-4582-9423-cd44a78500b7" />

and will be disabled, and if you toggled off the rentable should go back to the previous state

This behavior got changed in this pr; after toggling off the rentable, the old status doesn't go back